web/viewer.html

```diff
@@ -1,4 +1,4 @@
-﻿<!DOCTYPE html>
+<!DOCTYPE html>
```

Comment:
We should revert this change. I assume it removed the UTF-8 BOM, but as mentioned in #18370 if we do that we should probably do it outside of this patch in case something breaks (we don't _expect_ it to, but it would be a shame if we'd have to fix it or back out this patch because of it).